### DISCUSSION TITLE
Question on CALLOUT table information in MIMIC-IV

### DISCUSSION THREAD

**Discussion by riarungta** [MIT-LCP/mimic-code#1147] [MIMIC-IV]
Hello,

I am wondering if the information that was part of the CALLOUT table in MIMIC-III is available in MIMIC-IV? If it is, I would love to know which module it is part of since I am having trouble locating the variables that were part of this table.

Thanks1

**Comment by briangow** [comment]
Unfortunately, this information isn't in MIMIC-IV even though it is theoretically possible to add it. The callouts table was added to MIMIC-III as part of a particular project that needed this data. If there is a need for this data in MIMIC-IV, we can make a note to add it in a future release.

**Comment by riarungta** [comment]
Hi Brian,

Thank you for the information. I am interested in investigating potential
delays in discharge from the ICU and factors associated with this outcome.
Is it possible to obtain or calculate delays in discharge using any other
data that is already present in MIMIC-IV? If not, would you know the
estimated timeline of a potential future release?

Thank you!

Ria

Best,
*Ria Rungta, *MPH

**Comment by briangow** [comment]
As far as I know there isn't any other way to calculate delays in discharge from the ICU. To my knowledge, callouts is only place that indicates when a subject is ready to be discharged from the ICU. 

We will note that there is interest in having this table released in MIMIC-IV (see https://github.com/MIT-LCP/mimic-code/issues/1152). Unfortunately, we can't provide a timeline for when that will happen though.

**Comment by SichengH** [comment]
The callout table stopped updating from the source(the hospital information system) since 2018, to ensure consistency, we decide not to include partially available data in the MIMIC_IV.
